pootle/settings/90-local.conf.template

```diff
@@ -237,3 +237,6 @@ POOTLE_MT_BACKENDS = [
 TEMPLATES[0]['DIRS'] = [
     # working_path(os.path.join('custom', 'templates')),
 ]
+
+# True to always remember user session. False to not remember. None to ask user.
+ACOUNT_SESSION_REMEMBER = True
```

Comment:
We're already setting this is 40.  If we don't expect people to change it then lets leave it out.  If we do then lets comment it out so that they get the default we've configured.